### ISSUE TITLE
Background cut off on mobile

### ISSUE THREAD

**Issue by opncow** [jpanther/congo#285]
### What happened?

`baseof.html` sets `h-screen` class on `body`. On my phone, this has the effect that the background is cut off and the background color changes: 

![image](https://user-images.githubusercontent.com/1069651/187624868-c79c3076-05c9-4dc9-88c0-9b6c54614f8c.png)

I fixed it on my site by setting the body height to 100%:

```CSS
body {
    height: 100% !important;
}
```

### Theme version

v2.3.1

### Hugo version

hugo v0.101.0+extended linux/amd64

### What browsers are you seeing the problem on?

[Fennec](https://f-droid.org/packages/org.mozilla.fennec_fdroid/)

**Comment by jpanther** [comment]
Thanks for the report, @opncow. I've been trying to replicate this but haven't had any success. It looks like you're using an Android browser and I wonder if that is related. I don't have access to an Android device so it's difficult for me to test.

The fix you have proposed will not support the footer being at the bottom of the viewport on pages which have content that doesn't fill the whole screen.

This is the first report I've received about an issue like this and so I wonder if it's specific to a certain browser version?

**Comment by opncow** [comment]
Hi @jpanther, thanks for checking. You're right, it's browser-specific. I saw it on [Fennec](https://f-droid.org/packages/org.mozilla.fennec_fdroid) that I use as my only browser on my phone. It never behaved differently than the regular Firefox until now. To check it again, I now installed the regular Firefox and there I cannot reproduce the issue as well. Also Brave shows your page just fine.

Will remove the "fix" from my page again.

Thanks for the great template btw - it's really fun to work with!
